src/test/samVisualize/graphGeneration/graphGeneration.test.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as assert from 'assert'
+import { generateGraphFromYaml } from '../../../samVisualize/graphGeneration/cfnTemplateGraphGenerator'
+import { Node, Link } from '../../../samVisualize/graphGeneration/graph'
+import { join } from 'path'
+import { readdirSync, readFileSync } from 'fs'
+import { getProjectDir } from '../../testUtil'
+
+const TestInputDirectory = join(getProjectDir(), 'src', 'test', 'samVisualize', 'testYamlTemplates')
+const TestExpectedOutputDirectory = join(getProjectDir(), 'src', 'test', 'samVisualize', 'expectedGraphObjects')
```

Comment:
Need to remove `src` since `getProjectDir()` refers to the code source directory.